### ISSUE TITLE
bropy Error processing log file Ubuntu 16.04

### ISSUE THREAD

**Issue by jinverar** [hashtagcyber/bropy#13]
Hello there I'm hoping you can help me with an issue. I've install bropy correctly from what I can tell and I don't receive any errors within reporter.log. My notices.log looks good and I see baseline events inside the notices.log file

The issue is when I run bropy from command line and use option 1 or option 2 i get errors as displayed in the screenshot. I am try to baseline my bro files and I receive errors that lead me to believe something is wrong with parsing the notices.log.gz or notices_alarm.log.gx files.  I've tried a whole bunch of things to fix this with no success. 

Traceback "./bropy.py, line164, in <module>
menu()
File "./bropy.py", line 130, in menu
stepupdate()
file "./bropy.py", line 63, in stepupdate
for x in addbase

See the attachment. Please provide advice because the bropy tool looks AMAZING. 

![bropy-trouble](https://user-images.githubusercontent.com/926688/51435206-6933d500-1c40-11e9-986b-c6831a81d7ed.JPG)

![bropy-trouble2](https://user-images.githubusercontent.com/926688/51435237-27eff500-1c41-11e9-9281-2727d97fd098.JPG)
 
Thank you in advance for the support

~J

**Comment by jinverar** [comment]
I will try to let the bropy tool run for a day and then try again and see what happens.

**Comment by jinverar** [comment]
It looks like bropy errors are from the latest days files and may read yesterdays files correctly. I'm still trouble shooting.

**Comment by jinverar** [comment]
After troubleshooting for a while I think my issue is that the new security onion uses BRO with JSON format however I havn't confirmed this 100%....it's just a theory.

**Comment by hashtagcyber** [comment]
Great catch! When I wrote it, bro logged in a tab separated format.
Security onion uses a plugin to convert everything to JSON then push it
into ElasticSearch... If you wanted to use seconion with bropy you’ll need
to disable the json logs plugin (which will probably break multiple things
in SecOnion).

I’m actually working on a GUI version that runs as a VMWare appliance right
now, so I don’t think I’ll have time to update this one for a while. If
anyone wants to submit a PR that introduces JSON support, I’d be more than
happy to merge it.

**Comment by weslambert** [comment]
HI @hashtagcyber @jinverar ,

Disabling `json-logs` should be fine.

You should be able to switch back to TSV without issue by following the instructions here:

https://github.com/Security-Onion-Solutions/security-onion/wiki/Bro#tsv

Thanks,
Wes

**Comment by jinverar** [comment]
Hey @weslambert @hashtagcyber 

While you guys were replying I was already testing and troubleshooting and @weslambert is correct. I commended out the json-logs line from local.bro and reset bro.  The next step I had to do was re-run the so-setup on the security onion which cleaned out all the logs. Security onion kept allmost all my custom settings and kibana works with the TSV files just the same as with the JSON files on a home network!

I've got bropy running and RITA running on the new security onion and I've added it to a guide that I'm working on. Thanks alot. 
 
Thank you for the reply

~j
